### PR TITLE
Add issue templates and update README

### PR DESCRIPTION
**Pull request by pauby**
* Added issue templates to make it easier to raise issue and clearer why type of issues should be raised and avenues for others;
* Updated README with notes on package types and when / who should raise issues;

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/kPackageRequest.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+- [ ] The package I am requesting does not already exist on https://chocolatey.org/packages;
+- [ ] There is no open issue for this package;
+- [ ] The issue title starts 'RFP - ';
+- [ ] The download URL is public and not locked behind a paywall / login;
+
+## Package Details
+
+Software project URL :
+Direct download URL for the software / installer :
```
Comment (AdmiringWorm) [inline .github/ISSUE_TEMPLATE/kPackageRequest.md]: Perhaps having a value for a summary of the software as well?

As well as having a section the user can input what he/she expects the package to do (outside of just installing the program)

File: .github/ISSUE_TEMPLATE/mMaintainerRequest.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+- [ ] I have followed the Package Triage Process and I do NOT want to become maintainer of the package;
+- [ ] There is no existing open maintainer request for this package;
+
+## Checklist
+
+- [ ] Issue title starts with 'RFM - '
+
+## Existing Package Details
+
+Exiting package URL:
```
Comment (AdmiringWorm) [inline .github/ISSUE_TEMPLATE/mMaintainerRequest.md]: Optionally the package source url?

File: .github/ISSUE_TEMPLATE/kPackageRequest.md (hunk 1 above)
Comment (pauby) [reply .github/ISSUE_TEMPLATE/kPackageRequest.md]: That's a good suggestion. Added.

File: .github/ISSUE_TEMPLATE/mMaintainerRequest.md (hunk 2 above)
Comment (pauby) [reply .github/ISSUE_TEMPLATE/mMaintainerRequest.md]: Yeah, that makes good sense. Added.

**Comment by pauby** [comment]
Cheers for doing this so quickly @AdmiringWorm

**Pull request merged**